docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Lack of unique IDs and Aria-labels on sections and categories hinders automation
+
+There is a lack of unique IDs applied to sections and categories, which hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
+
+.Support disconnected installations
+
+MTA does not support disconnected installations. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Cannot remove the Jira connection to a Migration Wave that is later removed 
+
+It is not possible to remove a Jira connection to a Migration Wave after that Wave is removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
+
```

Comment:
@HagayVider1 If you check the existing MTA documentation, you will see that "migration wave" is not a proper noun.
https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.2/html-single/user_interface_guide/index#mta-web-creating-migration-waves_user-interface-guide